README.md

```diff
@@ -13,11 +13,8 @@ $ flatpak-builder build --install --user ./org.ioquake3.IOQuake3.yaml
 Definitely!
 
 I just need to:
```

Comment:
After merging my fixes and submitting the Flatpak as a PR to Flathub, there is one more thing we need to do:

`- … [ask upstream whether they are interested in co-maintaining the Flatpak]`

In other words, just [create a ticket here](https://github.com/ioquake/ioq3/issues) about the Flatpak and upstream push rights to its repo (as described in the new PR template on Flathub).

Reply:
Also, when submitting the final PR to Flathub, it would be good to remove the README file or replace it with something useful to end-users.